core/src/main/java/discord4j/core/object/reaction/DefaultReaction.java

```diff
@@ -0,0 +1,54 @@
+package discord4j.core.object.reaction;
+
+import discord4j.common.util.Snowflake;
+import discord4j.core.GatewayDiscordClient;
+import discord4j.core.object.DiscordObject;
+import discord4j.discordjson.json.DefaultReactionData;
+
+import java.util.Optional;
+
+public final class DefaultReaction implements DiscordObject {
```

Comment:
Maybe a docs here for what is a DefaultReaction or point the the getDefaultReaction method where is used.